### ISSUE TITLE
[Feature Request] Allow to restore keyspaces in a live cluster

### ISSUE THREAD

**Issue by cscetbon** [instaclustr/instaclustr-esop#18]
In order for [CassKop](https://github.com/Orange-OpenSource/casskop) to start using the sidecar we would need to be able to restore all tables in a keyspace or a list of keyspaces in a live cluster.  The restore should be simple to implement after #17

**Comment by smiklosovic** [comment]
Yes this should be simple, we can look at this problem not as this one is the extension of the table restoration but that this one is the generalisation of the restoration and table case is just its specific case. 

This will be more straightforward if we restore from a snapshot we backed up. In other words, we would extend the logic in table case to all tables in a keyspace / in all tables in a snapshot we backed up. It would be quite problematic to restore from two snapshots in one run. It would be just easier to do two restores in a row, each per keyspace but just in case our snapshot we restore from does not contain tables from both keyspaces already.
